documents/operating.md

```diff
@@ -11,12 +12,83 @@ Contents:
+* "Operating Agreement" shall refer to the official version of this document
+to be held in a file named `operating.md` in the `documents` directory of the
+`corporate` repository within the `BadIdeaFactory` team hosted on GitHub.
+
+### Support Categories
+* "**Complete Consensus**" shall mean 100% support or more.
+* "**Strong Consensus**" shall mean 2/3 support or more.
+* "**Majority**" shall mean over 1/2 support.
+* "**Apathetic Support**" shall mean over 1/3 vocalized support and less than 1/3 vocalized dissent.
+
+---------------------------------------------------------------------
 SECTION 1: 📚📚📚 Official Operating Agreement 📚📚📚
 ---------------------------------------------------------------------
-- [ ] Company term
-- [ ] Continuance of company
-- [ ] Rickroll
-- [ ] Game of Mario Party 3 before legal arbitration between members
+- [x] Company term
+- [x] Continuance of company
+- [x] Rickroll
+- [x] Game of Mario Party 3 before legal arbitration between members
+
+### Company Term
+BIFFUD will exist until either (1) a vote to terminate the organization
+is passed by a consensus of all Corporate Overlords or (2) the Hive Mind fails to
+hold a single meeting with quorum for a period of (a consecutive year?),
+(3) everyone dies / there are no more Corporate Overlords, or (4) if all
+Corporate Overlords are straight white cis men.
+
+### Continuance of Company
+If a Corporate Overlord leaves BIFFUD, dies, or uploads their brain to the
+cloud, the ownership of the remaining Corporate Overlords will then be evenly
+distributed (e.g. if four Corporate Overlords remain, each will own 1/4 of BIFFUD).
+
+### Lawsuits Between BIFFUD Associates
+All BIFFUD associates involved in a lawsuit must play a group game of
+Mario Party 3 before any arbitration happens.  BIFFUD will loan any
+necessary hardware needed to make this possible.  Efforts must be made
+to obtain a Nintendo 64, however if obtaining such hardware would cause
+unreasonable burdan, emulation is acceptable so long as a legal license
+for Mario Party 3 is first obtained by the Company.
+
+If more than four associates are involved in a lawsuit, a Mario Party
+elimination tournament must be played.  The terms of the tournament shall
+be dictated by a neutral thrid party.
+
+### Additional Notes
+It is worth explicitly stating that the Corporate Overlords will work
+tirelessly to avoid any of the following:
+
+- Giving up on their stakeholders.
+- Letting anybody down to whom they have made a commitment.
+- Disappearing / "running around" for long periods of time without communicating.
+- Deserting one another as members without formally abdicating their stake.
+
+Furthermore, they will try not to:
+
+- Make anybody cry.
+- Close the company and say "goodbye", so long as there someone is out there
+willing to carry the torch forward.
+
+Changes to Section 0 or Section 1 of this Operating Agreement will require a
```

Comment:
Legally! But from a sociological perspective, it might depend on whether this line of computer-mediated communication is considered as part of an oral or written culture.